src/core/Table/Filters/FilterButtonBar.tsx

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import '@itwin/itwinui-css/css/table.css';
+import { useTheme } from '../../utils/hooks/useTheme';
+import { Button } from '../../Buttons';
+
+export type FilterButtonBarProps = {
+  children?: React.ReactNode;
+  setFilter: () => void;
+  clearFilter: () => void;
+};
+
+export const FilterButtonBar = (props: FilterButtonBarProps) => {
```

Comment:
Description, what it is intended to be used for

Reply:
Added